docs/sources/operations/zone-ingesters.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Zone Aware Ingesters
+description: Migration guide and additional details
+weight: 10
+---
+
+# Zone Aware Ingesters
+Loki's zone aware ingesters are used by Grafana Labs in order to allow for easier rollouts of large Loki deployments. You can think of them as three logical zones, however with some extra k8s config you could deploy them in separate zones.
+
+By default, an incomming log stream's logs are replicated to 3 random (but the same 3, except in the case of some replica scaling) ingesters. This means that if one ingester is restarted, no data is lost, but two can mean data is lost and also impacts the systems ability to ingest logs because of an unhealthy ring status.
+
+With zone awareness enabled, an incomming log line will be replicated to one ingester in each zone. This means that we're no only concerned if ingesters in multiple zones restart at the same time. We can now rollout, or lose, an entire zone at once and not impact the system. This allows deployments with a large number of ingesters to be deployed too much more quickly.
+
+We also make use of [rollout-operator](https://github.com/grafana/rollout-operator) to manage rollouts to the 3 statefulsets gracefully. The rollout-operator looks for labels on statefulsets to know which statefulsets are part of a certain rollout group, and coordinate rollouts of pods only from a single statefulset in the group at a time. See the README in the rollout-operator repo. for a more in depth explanation.
+
+## Migration
+Migrating from a single ingester statefulset to 3 zone aware ingester statefulsets. The migration follows a few general steps, regardless of deployment method.
+0. configure your existing ingesters to be part of a zone, for example `zone-default`, this will allow us to later exclude them from the write path while still allowing for graceful shutdown
+1. Prep for the increase in active streams (due to the way streams are split between ingesters) by increasing the # of active streams allowed for your tenants
+2. add and scale up your new zone aware ingester statefulsets such that each has 1/3 of the total # of replicas you want to run
+3. enable zone awareness on the write path by setting `distributor.zone-awareness-enabled` to true for distributors and rulers
+4. wait some time to ensure that the new zone aware ingesters have data for the time period they are queried for (`query_ingesters_within`)
+5. enable zone awareness on the read path by setting `distributor.zone-awareness-enabled` to true for queriers
+6. configure distributors and rulers to exclude ingesters in the `zone-default` so those ingesters no longer receive write traffic via `distributor.excluded-zones`
+7. use the shutdown endpoint to flush data from the default ingesters, then scale down and remove the associated statefulset
+8. clean up
```

Comment:
This list could use a few more specific details in the steps, and there's some cleanup to be done re: capitalization and punctuation.

Reply:
This is the "general steps" section, so that people can do a migration without using the deployment methodologies we provide code (jsonnet) for.